pack-sdk.ps1

```diff
@@ -0,0 +1,13 @@
+# FOR TESTING - WILL DELETE
+
+$packageSuffix = "dev" + [datetime]::UtcNow.Ticks.ToString()
```

Comment:
If you are curious, for local builds the dotnet team likes to version everything as "42.42.42.42". I find that fun 😄

Reply:
hahaha love that :D